flame/star/star_model_tester.py

```
@@ -164,20 +166,21 @@ def write_result(result: Any,
 
             for i, res in enumerate(result):
-                if isinstance(result_filepath, list):
-                    current_path = result_filepath[i]
+                if isinstance(filename, list):
+                    current_path = filename[i]
                 else:
-                    if '.' in result_filepath:
-                        result_filename, result_extension = result_filepath.rsplit('.', 1)
-                        current_path = f"{result_filename}_{i + 1}.{result_extension}"
+                    if '.' in filename:
+                        result_filename, result_extension = filename.rsplit('.', 1)
```

Ruff (0.15.13)

[warning] Unpacked variable `result_filename` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)